tests/dbt/test_graph.py

```diff
@@ -251,6 +251,30 @@ def test_load_via_dbt_ls_with_invalid_dbt_path():
     assert err_info.value.args[0] == expected
 
 
+@pytest.mark.integration
+@pytest.mark.parametrize("pipeline_name", ("jaffle_shop", "jaffle_shop_python"))
```

Comment:
Thanks for adding the test, @DanMawdsleyBA !
```suggestion
@pytest.mark.parametrize("pipeline_name", ("jaffle_shop"))
```
Since `jaffle_shop_python` doesn't have any `packages.yml` - which is also a valid use-case